app/routes/goal.py

```diff
@@ -0,0 +1,104 @@
+from flask import Blueprint, jsonify, request, abort, make_response
+from app import db
+from app.models.goal import Goal
+from app.routes.task import get_task_from_id
+
+
+goal_bp = Blueprint("goal", __name__, url_prefix="/goals")
+
+
+@goal_bp.route('', methods=['POST'])
+def create_one_goal():
+    request_body = request.get_json()
+    try:
+        new_goal= Goal(title=request_body['title'])
+    except KeyError:
+        return jsonify({"details": "Invalid data"}), 400 
```

Comment:
Nice work with try-except.  However it would be good to let the user know what data is invalid.